be/src/util/broker_storage_backend.cpp

```diff
@@ -72,14 +89,14 @@ Status BrokerStorageBackend::download(const std::string& remote, const std::stri
     constexpr size_t buf_sz = 1024 * 1024;
     char read_buf[buf_sz];
```

Comment:
Do not use array to store buffer. Use a vector please. Array will allocate the buffer on stack and stack is very small.